### PR TITLE
[WIP] fix: Simplify selectors in toolbar component

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/1850

## Description
There is simplify of toolbar component selector. No visual changes predicted

### CODE REVIEW

File: src/toolbar.scss [hunk 1]
```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
...
-            margin: 0.625rem 0 0.125rem 0;
-          }
-        }
-        .#{$block}__overflow__form-label,
-        .#{$block}__overflow__label {
-          overflow: hidden;
-          white-space: nowrap;
-          text-overflow: ellipsis;
-        }
+  &__overflow__body {
```
Comment (jbadan) [inline src/toolbar.scss:144]: We shouldn't have 2 block levels in a name - http://getbem.com/naming/

File: src/toolbar.scss [hunk 2]
```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
...
+    @include fd-reset();
+
+    display: flex;
+    flex-direction: column;
+    max-width: 20rem;
+    padding: 0.1875rem 0.375rem;
+
+    > *,
+    > .#{$fd-namespace}-button {
```
Comment (jbadan) [inline src/toolbar.scss:153]: This needs to be removed - there should be no universal selectors (`*`) ever and we should not be using `.#{$fd-namespace}-*` as discussed previously.

File: src/toolbar.scss [hunk 3]
```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
...
+    }
+
+    > .#{$block}__separator {
+      height: $toolbar-overflow-separator-height;
+      margin: 0.375rem 0.1875rem;
+      width: auto;
+      background-color: var(--sapToolbar_SeparatorColor);
+    }
+
+    .#{$block}__overflow__label {
```
Comment (jbadan) [inline src/toolbar.scss:170]: Same comment as above

**Review by jbadan — CHANGES_REQUESTED** [review]
See comments

File: src/toolbar.scss [hunk 4]
```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
...
 
-    .#{$block}__overflow {
-      &.#{$fd-namespace}-popover {
-        .#{$block}__overflow__body {
-          padding: 0.25rem 0.5rem;
-        }
+    .#{$block}__overflow__body {
+      padding: 0.25rem 0.5rem;
+
+      .#{$block}__separator {
```
Comment (katekozlowska) [inline src/toolbar.scss:200]: missing `fd-reset()`

File: src/toolbar.scss [hunk 5]
```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
...
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    }
   }
 
   &--cozy {
     height: $toolbar-height-cozy;
+
     .#{$block}__separator {
```
Comment (katekozlowska) [inline src/toolbar.scss:193]: missing `fd-reset()`

**Review by katekozlowska — CHANGES_REQUESTED** [review]

**Pull request closed without merging**